### PR TITLE
+sink_schema

### PR DESCRIPTION
**Pull request by alitrack**
add create to DuckDB Schema support.

### CODE REVIEW

File: postgres_scanner.cpp [hunk 1]
```diff
@@ -939,7 +942,7 @@ AND table_type='BASE TABLE'
 		dconn
 		    .TableFunction(data.filter_pushdown ? "postgres_scan_pushdown" : "postgres_scan",
 		                   {Value(data.dsn), Value(data.source_schema), Value(table_name)})
-		    ->CreateView(table_name, data.overwrite, false);
+		    ->CreateView(data.sink_schema,table_name, data.overwrite, false);
```
Comment (Mause) [inline postgres_scanner.cpp:945]: Can you fix the formatting here please? You should just be able to run `make format`

File: README.md [hunk 2]
```diff
@@ -28,7 +29,16 @@ SELECT * FROM POSTGRES_SCAN('', 'public', 'mytable');
...
 `POSTGRES_SCAN` takes three string parameters, the `libpq` connection string (see above), a Postgres schema name and a table name. The schema name is often `public`.
 
+### `sink_schema` usage
 
+attach Postgres schema to another DuckDB schema.
+
+```sql
+-- create a new schema in DuckDB first
+create schema abc;
+CALL postgres_attach('dbname=postgres user=postgres host=127.0.0.1',source_schema='public' , sink_schema='abc');
```
Comment (Mause) [inline README.md:39]: Clean up the formatting in this sql as well?

File: postgres_scanner.cpp (hunk 1 above)
Comment (alitrack) [reply postgres_scanner.cpp:945]: got it

Comment (alitrack) [reply postgres_scanner.cpp]: done.

Comment (alitrack) [reply postgres_scanner.cpp]: @Mause any chance to merge PR?

**Comment by Mytherin** [comment]
Thanks! LGTM

**Pull request merged**